halyard-deploy/src/main/java/com/netflix/spinnaker/halyard/deploy/spinnaker/v1/service/OrcaService.java

```diff
@@ -39,6 +40,8 @@
 public abstract class OrcaService extends SpringService<OrcaService.Orca> {
   @Autowired OrcaProfileFactory orcaProfileFactory;
 
+  @Autowired PluginProfileFactory pluginProfileFactory;
```

Comment:
Small nit, use constructor based injection.

Reply:
Abstract class, cannot use constructor based injection. Get the following error in multiple classes:
```
/halyard-deploy/src/main/java/com/netflix/spinnaker/halyard/deploy/spinnaker/v1/service/OrcaBootstrapService.java:32: error: constructor OrcaService in class OrcaService cannot be applied to given types;
@Data
^
  required: OrcaProfileFactory,PluginProfileFactory
  found: no arguments
  reason: actual and formal argument lists differ in length
```